x/merkledb/db.go

```diff
@@ -1263,12 +1274,15 @@ func (db *merkleDB) getNode(key Key, hasValue bool) (*node, error) {
 	switch {
 	case db.closed:
 		return nil, database.ErrClosed
-	case key == Key{}:
```

Comment:
you still need this for the root since you are keeping a copy of it in memory separate from the dbs

Reply:
updated